src/main/controllers/RespondentApplicationDetailsController.ts

```diff
@@ -52,9 +52,66 @@ export default class RespondentApplicationDetailsController {
       }
     }
 
+    let claimantResponseDocDownload = undefined;
+
+    if (selectedApplication.value?.respondCollection && selectedApplication.value?.respondCollection.length) {
+      for (let i = selectedApplication.value?.respondCollection.length - 1; i >= 0; i--) {
+        if (
+          selectedApplication.value?.respondCollection[i].value.from === 'Claimant' &&
+          selectedApplication.value?.respondCollection[i].value.supportingMaterial !== undefined
+        ) {
+          claimantResponseDocDownload =
+            selectedApplication.value?.respondCollection[i].value.supportingMaterial[0].value.uploadedDocument;
+        }
+      }
+    }
+
+    if (claimantResponseDocDownload) {
+      try {
+        await getDocumentAdditionalInformation(claimantResponseDocDownload, req.session.user?.accessToken);
+      } catch (err) {
+        logger.error(err.message);
+        return res.redirect('/not-found');
```

Comment:
Why do we redirect to "not-found" if an error occurred during processing?

Reply:
Spoke to Mehmet. This is what we have for now. Not being able to find documents is a bug and he's currently working on it.